### PR TITLE
Lab 1

### PR DESCRIPTION
**Pull request by Mahbubk8**
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
Adding code
## What did you learn from the changes you have made?
New learning
## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
May be
## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
New learnings
## How were these changes tested?
Not at all
## A reference to a related issue in your repository (if applicable)
N/A
## Checklist
- [X ] I can confirm that my changes are working as intended

### CODE REVIEW

File: 01_materials/labs/lab_1.ipynb
Comment (tianyi21) [inline 01_materials/labs/lab_1.ipynb:1]: - b: Please always reinitialize your model with `model = Sequential() ...` to reset all model parameters in addition to `model.compile()`. Or, you are continuing the training from the previous trained state with a different model setup. You may check this with the accuracy of the first epoch.

**Review by tianyi21 — APPROVED** [review]
Hi Mahbub!

Thank you for submitting your DL assignments.

Your Lab1 is now complete.

Thanks,
Tianyi [LS]
